plugins/automation/spec/services/discourse_automation/destroy_spec.rb

```diff
@@ -34,10 +33,17 @@
       it { is_expected.to fail_to_find_a_model(:automation) }
     end
 
-    context "when user can't destroy the automation" do
-      fab!(:user) { Fabricate(:user) }
-
-      it { is_expected.to fail_a_policy(:can_destroy_automation) }
+    context "when everything's ok" do
```

Comment:
I added a context for the happy path (as we usually do this), but I moved it after the failing examples. That’s something I can move back, no worries. Here also, I was wondering if we should decide on which order we should enforce. (happy path first then failing examples, or possible failing examples, then at the end you have the happy path).

Reply:
:+1: No strong opinions on this